### PR TITLE
fix: expect result is null but got zero

### PR DESCRIPTION
**Pull request by r253hmdryou**
fix: expect result is 0 but got NaN #13422

This is wrong
Should return Null instead of Zero or NaN

```
MariaDB [test]> SELECT MAX(users.id) FROM users;
+---------------+
| MAX(users.id) |
+---------------+
|          NULL |
+---------------+
1 row in set (0.010 sec)

MariaDB [test]> INSERT INTO users(id, name) VALUE(0, "name");
Query OK, 1 row affected (0.014 sec)

MariaDB [test]> SELECT MAX(users.id) FROM users;
+---------------+
| MAX(users.id) |
+---------------+
|             0 |
+---------------+
1 row in set (0.010 sec)
```

### CODE REVIEW

File: test/integration/model/notExist.test.js [hunk 1]
```diff
@@ -41,18 +42,22 @@ describe(Support.getTestDialectTeaser('Model'), () => {
...
       await expect(this.Order.max('sequence', { where: { type: 'C' } })).to.eventually.be.equal(0);
       await expect(this.Order.min('sequence', { where: { type: 'C' } })).to.eventually.be.equal(0);
-
-      // DataTypes.DECIMAL or DataTypes.FLOAT: previous and PR#13422 both use `to.eventually.be.equal(0)`
       await expect(this.Order.sum('amount', { where: { type: 'C' } })).to.eventually.be.equal(0);
       await expect(this.Order.max('amount', { where: { type: 'C' } })).to.eventually.be.equal(0);
       await expect(this.Order.min('amount', { where: { type: 'C' } })).to.eventually.be.equal(0);
     });
+
+    it('should type not exist', async function() {
```
Comment (sdepold) [inline test/integration/model/notExist.test.js:54]: Could you please give that a more sensible title? Must have slipped through earlier :)

**Review by sdepold — APPROVED** [review]
Nice. Thank you!

**Review by sdepold — CHANGES_REQUESTED** [review]
Hm. Seems to be broken everywhere somehow

File: test/integration/model/notExist.test.js (hunk 1 above)
Comment (r253hmdryou) [reply test/integration/model/notExist.test.js]: I'm sorry, I'm non-native so I can't think of proper English...

**Comment by r253hmdryou** [comment]
@sdepold 
I was misunderstanding
This time it should move firmly

**Review by sdepold — APPROVED** [review]

**Pull request merged**